service/worker/replicator/replicationTask.go

```diff
@@ -433,43 +297,21 @@ func (t *historyMetadataReplicationTask) Execute() error {
 			t.version,
 			t.nextEventID,
 			t.version)
-	} else {
```

Comment:
check on line 291 does not make sense: `if t.version != common.EmptyVersion {` probably should remove